app/controllers/concerns/blacklight/suggest/suggest_helper.rb

```diff
@@ -0,0 +1,32 @@
+module Blacklight
+  module Suggest
+    module SuggestHelper
```

Comment:
I'd say either put it in one place or make it a class that gets instantiated. IMHO a mixin just makes it harder to read.